litmus-portal/graphql-server/graph/schema.resolvers.go

```
@@ -599,12 +600,16 @@ func (r *subscriptionResolver) ViewDashboard(ctx context.Context, promQueries []
 		<-ctx.Done()
 		log.Printf("Closed dashboard view %v\n", viewID.String())
 		if _, ok := data_store.Store.DashboardData[viewID.String()]; ok {
+			err := ops.UpdateViewedAt(dashboardID, viewID.String())
+			if err != nil {
+				log.Printf("error - %v\n", err)
```

Can you return the err?